simple_ldap_user/simple_ldap_user.admin.inc

```diff
@@ -295,10 +295,18 @@ function simple_ldap_user_import($form, &$form_state) {
   asort($users);
 
   // Only selects new users
-  $drupal_users = entity_load('user');
+  
   $new_names = $users;
-  foreach ($drupal_users as $user) {
-      unset($new_names[strtoupper($user->name)]);
+  $sql="select name from {users} where name= :name";
+  foreach($new_names as $item){
+     $result=db_query($sql, array(':name'=>$item));
+     if($result)
+     {
+        while($row=$result->fetchAssoc()){
+            #Filter users fetched from ldap database and mysql database
```

Comment:
The Drupal coding standards discourage the use of perl-style comments.  Change this to a // comment.

https://www.drupal.org/docs/develop/coding-standards/api-documentation-and-comment-standards#inline